frontend/src/components/datasets/DatasetTable.tsx

```diff
@@ -1,166 +1,70 @@
-                    (validation_status === ValidationStatus.PASSED && <Button style={{color: "#a0d911"}}><CheckOutlined onPointerEnterCapture={undefined} onPointerLeaveCapture={undefined}/>Passed</Button>)
-                    ||
-                    (validation_status === ValidationStatus.FAILED && <Button style={{color: "#f5222d"}}><CloseOutlined onPointerEnterCapture={undefined} onPointerLeaveCapture={undefined}/>Failed</Button>)
-                )
-            }
-        },
-        {
-            title: "Readsets Released",
-            dataIndex: "released_status_count",
-            sorter: true,
-            render: (released_status_count, dataset) => {
-                return `${released_status_count}/${dataset?.readset_count}`
-            }
-        },
-        {
-            title: "Latest Release Status Update",
-            dataIndex: "latest_release_update",
-            sorter: true,
-            render: (latest_release_update, _) => {
-                return latest_release_update ? moment(latest_release_update).format("YYYY-MM-DD LT") : ""
-            }
-        },
-    ].map((column) => ({ ...column, key: column.title }))
+export interface DatasetTableProps {
+	run_name?: ExperimentRun['name']
+	scroll?: NonNullable<PagedItemsTableProps<ObjectWithDataset>['scroll']>
 }
+function DatasetTable({ run_name, scroll }: DatasetTableProps) {
 
-interface DatasetTableProps {
-    run_name?: FMSDataset['run_name']
-}
-
-const DatasetTable = ({ run_name }: DatasetTableProps) => {
-    const datasetsState = useAppSelector(selectDatasetsState)
-
-    const datasets = datasetsState.items
-    const filteredDatasets = datasetsState.filteredItems
-    const datasetsById = datasetsState.itemsByID
-    
-    const totalFilteredCount = datasetsState.filteredItemsCount
-    const totalCount = datasetsState.totalCount
-
-    const filters = datasetsState.filters
-    const isFetching = datasetsState.isFetching
-    const sortBy = datasetsState.sortBy
-    const page = datasetsState.page
+	const pagedItems = useAppSelector(selectDatasetsTable)
+	const { filters } = pagedItems
 
-    const dispatch = useAppDispatch()
-    const dispatchSetFilter = useCallback((...args) => dispatch(setFilter(...args)), [dispatch])
-    const dispatchSetFilterOption = useCallback((...args) => dispatch(setFilterOption(...args)), [dispatch])
-    const dispatchClearFilters = useCallback((...args) => dispatch(clearFilters(...args)), [dispatch])
-    const dispatchListFilter = useCallback((...args) => dispatch(listFilter(...args)), [dispatch])
-    const dispatchListTable = useCallback((...args) => dispatch(listTable(...args)), [dispatch])
-    const dispatchSetSortBy = useCallback((...args) => dispatch(setSortBy(...args)), [dispatch])
+	const callbacks = usePagedItemsActionsCallbacks(DatasetsTableActions)
+	useEffect(() => {
+		if (run_name) {
+			callbacks.setFilterCallback(DATASET_FILTER_KEYS.RUN, DATASET_FILTER_DEFINITIONS[DATASET_FILTER_KEYS.RUN])
```

Comment:
The run is not getting filtered correctly ... All datasets get displayed in the dataset tab of the experiment run.